src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/sidebar/menu.html.twig

```diff
@@ -1,4 +1,4 @@
 <div class="collapse navbar-collapse" id="sidebar-menu">
-    {% include '@SyliusAdmin/shared/crud/common/sidebar/search.html.twig' %}
+    {% hook 'search' %}
     {{ knp_menu_render('sylius_admin.main', {'template': '@SyliusAdmin/shared/crud/common/sidebar/menu/menu.html.twig', 'currentClass': 'active'}) }}
```

Comment:
And then this menu could be moved to a separate hookable, let's name it `'sylius_admin.common.component.sidebar.menu.items'`

Reply:
This way we don’t break the RWD since both hookables are wrapped in the Bootstrap class.

Reply:
Good idea! I implemented it, and its working great 🙂

Reply:
Good catch. I found this entry: https://docs.sylius.com/the-customization-guide/customizing-templates#priority

I would be way too optimistic if I said it’s easy to find 🙂  
Also, we don’t have recommendations for Twig hooks written down in our docs yet, so it would be great to propose some.